wrapper/src/test/java/integration/container/tests/ReadWriteSplittingTests.java

```diff
@@ -65,7 +65,6 @@
 import software.amazon.jdbc.HikariPooledConnectionProvider;
 import software.amazon.jdbc.PropertyDefinition;
 import software.amazon.jdbc.hostlistprovider.AuroraHostListProvider;
-import software.amazon.jdbc.hostlistprovider.ConnectionStringHostListProvider;
```

Comment:
this PR doesn't remove any code - is it unused?

Reply:
Yes it's an unused import